tensorflow/core/platform/rocm_rocdl_path.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef THIRD_PARTY_TENSORFLOW_CORE_PLATFORM_ROCM_ROCDL_PATH_H_
```

Comment:
Use of THIRD_PARTY_ in inclusion guards detected.

Header inclusion guards must not be prefixed with THIRD_PARTY -- open source 
tensorflow is not contained in a third_party root directory.

Reply:
@rthadur Thanks. I've pushed a new commit to address this.